test/supplemental_genbank_tests/genbank_upload_full_test.py

```diff
@@ -32,25 +35,26 @@ def setUpClass(cls):
             cls.cfg[nameval[0]] = nameval[1]
         cls.wsURL = cls.cfg['workspace-url']
         cls.wsClient = workspaceService(cls.wsURL, token=token)
+        suffix = int(time.time() * 1000)
+        cls.wsName = "test_GenomeFileUtil_" + str(suffix)
+        cls.wsID = cls.wsClient.create_workspace({'workspace': cls.wsName})[0]
         cls.serviceImpl = GenomeFileUtil(cls.cfg)
+        cls.dfUtil = DataFileUtil(os.environ['SDK_CALLBACK_URL'])
 
     @classmethod
     def tearDownClass(cls):
         if hasattr(cls, 'wsName'):
             cls.wsClient.delete_workspace({'workspace': cls.wsName})
             print('Test workspace was deleted')
 
-    def getWsClient(self):
-        return self.__class__.wsClient
+    def getDfuClient(self):
+        return self.__class__.dfUtil
+
+    def getWsID(self):
+        return self.__class__.wsID
```

Comment:
I would just access these directly via self rather than a getter, it's not necessary. The `__class__` shouldn't be necessary either

Reply:
I updated this file and removed all getter methods.
But all other test files in this repo still follow this `getter, __class__` format  
Plus, should switch from unittest to pytest framework

Reply:
yeah, the syntax / test structure here is ancient

Reply:
👍

Reply:
:+1: